### PR TITLE
Feature/lp 72 filled text field

### PR DESCRIPTION
**Pull request by NomasChuy**
Hi, this is my current ticket:
https://devaptivist.atlassian.net/browse/LP-72?search_id=c6e022c4-6b19-4ffd-82f6-8e3d3eedb814
I created the DropdownTextField component and added a string for accessibility purposes

Heres an example result of the component:
image

### CODE REVIEW

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 1]
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:177]: Please, add the param text to the documentation and the modifier to the documentation and function.

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 2]
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
...
+ * @param items it refers to the list that will be given to expand the textfield
+ *
+ * @author Jesus Lopez Gonzalez
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun DropdownTextField(
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt:187]: I think you need a high order function to get the information of the text field that it is selected

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 3]
```diff
@@ -167,3 +173,64 @@ fun LpOutlinedTextFieldNumber(
...
+    title: String,
+    items: List<String>
+) {
+    var expanded by remember { mutableStateOf(false) }
+    var selectedOptionText by remember { mutableStateOf("") }
+
+    ExposedDropdownMenuBox(
+        expanded = expanded,
+        onExpandedChange = { expanded = !expanded },
+        modifier = Modifier.padding(all = 24.dp)
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: please, add the modifier of the params function

File: core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt [hunk 4]
```diff
@@ -167,3 +171,70 @@ fun LpOutlinedTextFieldNumber(
...
     )
 }
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
+ * @param title refers to the label that the textfield will have
+ * @param items refers to the list that will be given to expand the textfield
+ * @param modifier to set component modifier
+ * @param
```
Comment (melissaquezada94) [inline core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt]: You forget to add the param selected, please add the description for this parameter

**Review by melissaquezada94 — APPROVED** [review]

**Review by carlosph816 — APPROVED** [review]
Well done!

**Pull request merged**